packages/trees/backend/sirius-components-collaborative-trees/src/main/java/org/eclipse/sirius/components/collaborative/trees/api/TreeCreationParameters.java

```diff
@@ -37,6 +37,8 @@ public final class TreeCreationParameters {
 
     private IEditingContext editingContext;
 
+    private Object object;
+
```

Comment:
I find the term object a little too generic, perhaps targetObject would be a little better.

Reply:
Done